Vagrantfile

```diff
@@ -4,20 +4,36 @@ load "settings.sh"
 
 Vagrant.configure("2") do |config|
   
-  config.vm.box = "fujimakishouten/debian-buster64"
+  config.vm.box = VB_IMAGE
 	
-  config.vm.network "public_network", ip: LAN_IP, bridge: INTERFACE, netmask: "255.255.255.0"
+  if STANDALONE.downcase == "yes"		
+    config.vm.network "public_network", ip: LAN_IP, netmask: "255.255.255.0", bridge: LAN_IF
+  else
+    config.vm.network "public_network", auto_config: true, bridge: LAN_IF
+  end
+  
+  if Vagrant.has_plugin?("vagrant-cachier")
+    config.cache.scope = :box
+    config.vm.network :private_network, ip: "10.0.2.52"
```

Comment:
I think that network config of the box should not be different for the case of caching.

Reply:
No this is required. Otherwise it gives an error
```
NFS requires a host-only network to be created.
Please add a host-only network to the machine (with either DHCP or a
static IP) for NFS to work.
``` 
They added this feature but didnt mention about it on the readme or documentation. Strange

Maybe the reason has something to do with project is not being maintained anymore - https://github.com/fgrehm/vagrant-cachier#warning-unmaintained-warning

Reply:
> If this is the case, then maybe you should not use it at all. Either find some other way of caching or no caching at all.

I think removing caching is our only option.